### PR TITLE
Prune images on target cluster

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1868981

CP for 4.5, 4.6

### CODE REVIEW

**Review by sergiordlr — APPROVED** [review]
Looks good to me.

File: modules/migration-running-migration-plan-cam.adoc
Comment (kalexand-rh) [inline modules/migration-running-migration-plan-cam.adoc:22]: Consider breaking this step in half:

```
. Log in to the source cluster as a user with ______ permissions.
. Delete old images by using the following command:
```

**Review by kalexand-rh — COMMENTED** [review]
I have a suggestion, but this otherwise LGTM.

File: modules/migration-running-migration-plan-cam.adoc
Comment (apinnick) [reply modules/migration-running-migration-plan-cam.adoc:22]: Permissions step not necessary. Already covered in the entire doc.

Comment (kalexand-rh) [reply modules/migration-running-migration-plan-cam.adoc]: Then I'll merge it when Travis finishes. Thanks!

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Pull request merged**
